contracts/base/Loans.sol

```diff
@@ -232,6 +230,9 @@ contract Loans is Base, LoansInterface {
             request.duration,
             termsExpiry
         );
+        if (msg.value > 0) {
+            emit CollateralDeposited(loanID, request.borrower, msg.value);
+        }
```

Comment:
Because on setting of loan terms you can also send in collateral. Theres a fixed time before your last collateral deposit and being able to take out the loan, so we wanted to allow users to deposit their collateral when they submit their loan terms. That way the collateral is in as early as possible and the waiting to take out the loan can begin

Reply:
Exactly, as @hensha256 mentions. Thanks!